tests/test_dataflow.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
```

Comment:
Why this import if we don't change anything in that file? Was it missing previously? Just curious.